Resources/Prototypes/GameRules/cargo_gifts.yml

```diff
@@ -213,3 +218,78 @@
       SecurityRiot: 2
       SecurityRestraints: 2
       SecurityNonLethal: 2
+
+- type: entity
+  id: GiftsFunPlushie
+  parent: CargoGiftsBase
+  components:
+  - type: StationEvent
+    weight: 3
```

Comment:
it will make the likelihood of useful gifts falling out much less likely.

Reply:
These are more meant to be fun additions rather than specifically be useful; what should they be weighted instead?